wire-ios-request-strategy/Sources/Request Strategies/Conversation/MLSClientIDsProvider.swift

```diff
@@ -1,5 +1,6 @@
+////
```

Comment:
Are the four slashes actually intended? I have seen that in some files and was about to replace them with two slashes.

Reply:
No I think it's something wrong with the header generation.